bower.json

```diff
@@ -31,22 +31,25 @@
   ],
   "dependencies": {
     "firebase": "^2.1.0",
-    "ember-data": "^1.13.0"
+    "ember-data": "^2.0.0"
   },
   "devDependencies": {
-    "ember": "1.13.3",
+    "ember": "^2.0.0",
```

Comment:
fix all the `devDependencies` to a specific version, they're only used in the dummy app/tests (and ember-try will check against the latest version)